mmcv/runner/hooks/lr_updater.py

```diff
@@ -278,6 +278,58 @@ def get_lr(self, runner, base_lr):
         return annealing_cos(base_lr, target_lr, progress / max_progress)
 
 
+@HOOKS.register_module()
+class FlatCosineAnnealingLrUpdaterHook(LrUpdaterHook):
+    """Flat + Cosine lr schedule.
+
+    Modified from https://github.com/fastai/fastai/blob/master/fastai/callback/schedule.py#L128 # noqa: E501
+
+    Args:
+        start_pct (float): When to start annealing the learning rate
```

Comment:
1. pct is an unfriendly abbreviation. Suggest changing a name.

Reply:
What name is better in your opinion?

Reply:
How about flat_ratio=0.25?

Reply:
Maybe fine. But it is not very true. Because if a warmup stage is used, the flat ratio is not the given value.

Reply:
warmup will affect no matter what the name is. Do you know the full name of pct?

Reply:
`start_pct` full name `start_percent`, means the percentage of the whole training progress.

Reply:
then we should use start_percent rather than start_pct as it is really hard to guess.

Reply:
ok